frontend/pages/entities/[slug].page.tsx

```diff
@@ -38,6 +52,27 @@ const EntityDetailPage: FC<EntityProps> = ({ entity }: EntityProps) => {
       />
       <ModuleContainer>
         <ModuleDetail module={entity} />
+        {(entity.relatedEntities?.length ?? 0) > 0 && (
+          <>
+            <CardHeader
+              title={`Entitles related to ${entity.name}:`}
+              style={{ textAlign: "center" }}
+            ></CardHeader>
+            <Box sx={{ display: "flex", flexWrap: "wrap", justifyContent: "center" }}>
+              {entity.relatedEntities?.map((relatedEntity) => (
+                <Link href={`/entities/${relatedEntity.slug}`} key={relatedEntity.slug} passHref>
+                  <a>
+                    <EntityRelatedEntityCard raised sx={{ margin: "0.3rem" }}>
+                      <Typography sx={{ padding: "0.5rem", textAlign: "center" }}>
+                        <HTMLEllipsis unsafeHTML={relatedEntity.name} maxLine="1" basedOn="words" />
+                      </Typography>
+                    </EntityRelatedEntityCard>
+                  </a>
+                </Link>
```

Comment:
We don't need `passHref` here.
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag